src/main/fc/fc_rc.c

```diff
@@ -239,6 +237,105 @@ FAST_CODE FAST_CODE_NOINLINE void processRcCommand(void)
         rcInterpolationStepCount = 0; // reset factor in case of level modes flip flopping
     }
 
+    DEBUG_SET(DEBUG_RC_INTERPOLATION, 2, rcInterpolationStepCount);
+
+    return updatedChannel;
+
+}
+
+#ifdef USE_RC_SMOOTHING_FILTER
+FAST_CODE FAST_CODE_NOINLINE uint8_t processRcSmoothingFilter(void)
+{
+
+    uint8_t updatedChannel = 0;
+
+    static float lastRxData[4];
```

Comment:
Question for @DieHertz: Will static variables in `FAST_CODE` go into `FAST_RAM` automatically, or do they have to be attributed for this?

Reply:
They won't, and there's absolutely no relationship between the two.
And you may actually want them to go to `SRAM` sometimes rather than `DTCM-RAM`.

Reply:
So declaring `FAST_CODE` will work for static local variables, and will put them into fast RAM?

Reply:
Not sure I understand your last comment. `FAST_CODE` only affects instructions. Static variables will still go into SRAM unless marked `FASTRAM`

Reply:
Sorry, I meant to say `FAST_RAM` above.